TestBenches/FileReadUtility.h

```diff
@@ -4,6 +4,7 @@
 #ifndef TestBenches_FileReadUtility_h
 #define TestBenches_FileReadUtility_h
 
+#include <algorithm>
```

Comment:
@aperloff Is this extra include necessary?

Reply:
It is. I was surprised that no one else caught it, but `std::remove` comes from the `algorithm` header. Some of the unit tests were failing because it wasn't included in the FileReadUtility. `std::remove` is used on L85.

Reply:
That's good enough for me. But I am curious what unit tests you're talking about. I don't see any warnings or errors in the output from the CI tests.